jwst/ami/ami_analyze_step.py

```diff
@@ -32,8 +34,18 @@ def process(self, input):
         # Retrieve the parameter values
         oversample = self.oversample
         rotate = self.rotation
-        self.log.info('Oversampling factor = %d', oversample)
-        self.log.info('Initial rotation guess = %g deg', rotate)
+
+        # pull out parameters that are strings and change to floats
+
+        psf_offset = self.set_psf_offset_find_rotation.split(' ')
+        psf_offset_find_rotation = (float(psf_offset[0]), float(psf_offset[1]))
```

Comment:
Slightly more readable would be:

```python
psf_offset_find_rotation = [float(a) for a in self.set_psf_offset_find_rotation.split()]
```

Same below for `rotsearch_parameters`.

Reply:
I will make the change but it is not more readable to me. In fact I had to stare at that a while to understand what it was going